inference-engine/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -162,6 +162,10 @@ void SubgraphBaseTest::compile_model() {
     executableNetwork = core->compile_model(function, targetDevice, configuration);
 }
 
+void SubgraphBaseTest::init_ref_function(std::shared_ptr<ov::Function> &funcRef, const std::vector<ov::Shape>& targetInputStaticShapes) {
```

Comment:
Why do you add additional call to the callstack? It has the same signature and, to be honest, looks extra

Reply:
I decided that `resize_function` is utility function and should not be defined as class member. Should I replace `init_ref_function` with `resize_function`?

Reply:
add comments

Reply:
discussed offline